target/scripts/build/packages.sh

```diff
@@ -205,6 +191,10 @@ function _install_getmail() {
   apt-get "${QUIET}" autoremove
 }
 
+function _install_dovecot_fts_xapian_dependency() {
+  apt-get "${QUIET}" --no-install-recommends install libxapian30
+}
```

Comment:
Is this necessary for a single line?

Keeping isolation is fine, but I think that can just be handled as a separate line in `_install_dovecot` with adding the dependency to `DOVECOT_PACKAGES` array. It can have a comment above it to clarify it's a dependency specifically for the FTS Xapian support.